code/orders-boot/src/main/resources/application.yml

```
@@ -4,6 +4,10 @@ management:
       exposure:
         include: health, info, metrics, prometheus
 
+app :
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)